src/scss/attendee/effects/sparkle.scss

```diff
@@ -1,34 +1,24 @@
 @use "../theme";
 
-$-blur: 5px;
+$blur: 5px;
```

Comment:
[NOTE] Might be useful to check design if there are different blurs and shadows planned